client/blocks/nav-unification-modal/index.jsx

```diff
@@ -116,7 +116,7 @@ const Modal = () => {
 							}
 							heading={ translate( 'Do even more' ) }
 							content={ translate(
-								'Advanced admin features have a new home! You can find it in your account settings'
+								'Advanced admin features have a new home! You can find it in your Account Settings.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.